src/helpers/configHelpers.ts

```diff
@@ -16,7 +16,7 @@
 // ________________________________________________________________________________________________
 
 import type { Client, Guild, ThreadChannel } from "discord.js";
-import * as defaultConfig from "../config.json";
+import * as defaultConfig from "../../config.json";
```

Comment:
❗This is a breaking change, the next release to include this PR must be a major (eg. `3.0.0`) one.

Reply:
hope next release is v3 anyways 🙂